apps/condo/domains/contact/hooks/useImporterFunctions.tsx

```diff
@@ -210,7 +215,7 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
             }
 
             const contactData = {
-                organization: { connect: { id: String(userOrganizationId) } },
+                organization: { connect: { id: String(userOrganizationIdRef.current) } },
```

Comment:
Why here you need to wrap organizationId with `String()`?

Reply:
it doesn't seem to affect anything. 
if it matters i can remove it